docker/python-plotting/package.json

```diff
@@ -1,4 +1,4 @@
 {
     "name": "python-plotting",
-    "version": "1.2.0"
+    "version": "1.2.1"
```

Comment:
I wonder if these images that are decoupled from an underlying tool version should scrap SemVer and just be integer versioned.

Reply:
Can you elaborate on your reasoning?

Reply:
I'm just not convinced that SemVer's normal definitions apply to many of these changes. I don't know that many of our container updates will clearly fit that format. So it may be simpler to just increment an integer release. Even this PR, I'm not really sure it warrants a version bump.

Reply:
I'm not sure. I'd like to think about this some more. Can we postpone this decision? And merge this in the meantime? (we can continue the discussion over slack)

Reply:
Yes, I meant to mention that previously. I'm not sure it should've gone in this PR anyway. Fine to continue the discussion later.